clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStoragePartition.java

```diff
@@ -249,6 +270,46 @@ public RocksDBStoragePartition(
         Collections.singletonList(RocksDB.DEFAULT_COLUMN_FAMILY));
   }
 
+  private void checkMemoryLimit(long memoryLimit, SstFileManager sstFileManager, String dbPath) {
```

Comment:
So the intention of this PR is to check SST total file size during RocksDB#open(), it could not dynamically track it during hybrid ingestion when new SST files are being materialized continuously into the opened RocksDB? Would it be a concern>?

Reply:
Today, RocksDB SSTFileManager impl doesn't check space limit when opening up an existing database, and there is the reason we have this logic here.

when the database is opened, SSTFileManager will track every write and it is not a concern.

Reply:
Oh I see... I think I missed this line:

>         this.sstFileManager.setMaxAllowedSpaceUsage(this.memoryLimit);
Now I get how it is tracking the usage dynamically.